UbiquityOS-User-Manual/key-products/README.md

```diff
@@ -0,0 +1,37 @@
+# Key Products
+
+The Ubiquity ecosystem is a dynamic and innovative suite of decentralized finance (DeFi) products designed to redefine stability and utility in the blockchain space. At its heart lies the **Ubiquity Dollar (uUSD)**, a flexible and scalable stablecoin protocol built to serve as a cornerstone for DeFi applications.&#x20;
+
+Supporting this core is the **Yield Aggregator**, which empowers users by maximizing returns through efficient asset allocation. Unified by shared governance via the **UBQ token** and a sustainable **protocol treasury**, the ecosystem is designed for interoperability, enabling each product to thrive independently while benefiting from seamless integration and collective growth.
+
+## Ubiquity Ecosystem Products
+
+### **Ubiquity** **Financial** **Products**
+
+1. **Ubiquity Dollar (UUSD)**
+
+**The Ubiquity Dollar (UUSD)** is a stablecoin pegged to the US Dollar, designed to deliver financial stability and enable seamless transactions within decentralized finance (DeFi) applications and the evolving metaverse. It combines reliability and scalability, making it an ideal choice for both traditional and virtual economies.
+
+2. **Ubiquity Cards**
+
+Ubiquity Cards are designed to seamlessly connect the digital economy, particularly the growing metaverse, with traditional real-world financial transactions. Issued as virtual prepaid cards, they are compatible with Apple Pay, Android Pay, and Samsung Pay, ensuring ease of use in both digital and physical retail environments.
+
+3. **Ubiquity Wallet**
+
+The **Ubiquity Wallet** is a crucial element of the Ubiquity ecosystem, designed to integrate payments and governance infrastructure to improve user experience and drive adoption. As of March 2024, the development phase includes the creation of native iOS and Android apps, as well as a browser extension, to ensure broad accessibility and functionality.&#x20;
+
+
+
+### **Ubiquity** **Productivity** **Tools**
+
+1. **Ubiquity OS DevPool**
+
+The **Ubiquity OS DevPool** is an innovative platform that combines the power of decentralized finance (DeFi) and artificial intelligence (AI). It offers dynamic rewards, advanced governance tools, and global collaboration solutions, empowering contributors and organizations to thrive in a decentralized ecosystem.
+
+2. **Ubiquity Superintendent**
+
+The Ubiquity Superintendent, known as **UbiquiBot** or **The Kernel,** is an advanced AI agent designed to optimize and automate project management across the Ubiquity network. It integrates comprehensively with GitHub, supporting all events such as issue closures, pull requests, and new comments, providing real-time insights and automated responses for effective project management.&#x20;
```

Comment:
This is incorrect nomenclature. The superintendent actually just takes care of synchronising open issues for the frontend. You still use the term `ubiquibot` here which is also wrong.

The kernel is just what serves events while `ubiquity-os` is the whole thing, the kernel by itself alone is not really useful.